src/Symfony/Component/HttpKernel/Tests/EventListener/LocaleAwareListenerTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\EventListener;
+
+use ArrayIterator;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\FinishRequestEvent;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\EventListener\LocaleAwareListener;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Contracts\Service\LocaleAwareInterface;
+
+class LocaleAwareListenerTest extends TestCase
+{
+    private $listener;
+    private $localeAwareService;
+    private $requestStack;
+
+    protected function setUp()
+    {
+        $this->localeAwareService = $this->getMockBuilder(LocaleAwareInterface::class)->getMock();
+        $this->requestStack = $this->getMockBuilder('Symfony\Component\HttpFoundation\RequestStack')->getMock();
```

Comment:
`RequestStack::class`

Reply:
Actually, there is no reason to mock the request stack. Let's just use `new RequestStack()` here.